core/tests/karma-globals.js

```diff
@@ -17,7 +17,21 @@
  */
 
 var GLOBALS = {
-  INTERACTION_SPECS: {},
+  INTERACTION_SPECS: {
+    TextInput: {
+      customization_arg_specs: [
+        {},
```

Comment:
Could you just replicate the full dict? I think having just a partial one here is confusing.